### PR TITLE
add scheduler work with api server doc

### LINKED ISSUE #103: add a doc for  new scheduler work with api server
The doc is used for understanding how the global scheduler works with API Server, with a usage instruction and implementation process description.

### PR DESCRIPTION
**Pull request by nkwangjun**
**What type of PR is this?**

> /kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #103

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by nkwangjun** [comment]
Please merge to master, thanks. @cathyhongzhang

File: docs/design-proposals/global-scheduler/SchedulerWorkWithAPIServer.md [hunk 1]
```diff
@@ -36,11 +21,11 @@ export CONFIG_BASE=${PWD}/conf
 # kubeconfig: config of the API Server
 # scheduler_policy_file: the scheduler policy file path, the default file path is also in ${PWD}/conf/conf.yaml
 
-# run gs-scheduler
-./_output/bin/gs-scheduler
+# build and run
+hack/gs-scheduler-up.sh
```
Comment (jshaofuturewei) [inline docs/design-proposals/global-scheduler/SchedulerWorkWithAPIServer.md:25]: The script does not come with the pull request #101

Comment (cathyhongzhang) [reply docs/design-proposals/global-scheduler/SchedulerWorkWithAPIServer.md:25]: We an not find the hack/gs-scheduler-up.sh in the code of PR 101

Comment (jshaofuturewei) [reply docs/design-proposals/global-scheduler/SchedulerWorkWithAPIServer.md:25]: The script is at https://github.com/futurewei-cloud/global-resource-scheduler/blob/a906940f50c0336d7efa9f212ae4b162baa100b5/hack/gs-scheduler-up.sh which is at #101.

**Review by cathyhongzhang — APPROVED** [review]

**Review by jshaofuturewei — APPROVED** [review]

**Pull request merged**